src/components/backend-ai-session-launcher.ts

```diff
@@ -2214,8 +2245,11 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
     return false;
   }
 
```

Comment:
Great! It's better to use [decorators](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#documentation) not only for functions or variables, but also in comments, too.

Reply:
We already have a similar issue raised in #1376, but it's a bit different (since is documentation specific.)
I'll just generalize the title and add this feature as a partial task.

Reply:
👍